libs/design-system/web-component-card/src/lib/web-component-card.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { WebComponentCardComponent } from './web-component-card.component';
+import { render, screen } from '@testing-library/angular';
+
+describe('WebComponentCardComponent', () => {
+  it('Error should be visible in the indicator', async () => {
```

Comment:
This text could be improved. It should read as a complete sentence, for example: `it('should display the provided title, component name and description', ...)`
(Feel free to use the example text as is)